db/migrate/20160626235137_create_join_table_category_keyword.rb

```diff
@@ -0,0 +1,8 @@
+class CreateJoinTableCategoryKeyword < ActiveRecord::Migration
+  def change
+    create_join_table :Categories, :Keywords do |t|
```

Comment:
Ah, `ActiveRecord` doesn't normalize them when creating the join table. These should def be lower case.